data/presets/power/generator/source/wind.json

```diff
@@ -4,15 +4,14 @@
         "ref",
         "operator",
         "generator/type",
-        "generator/output/electricity"
+        "generator/output/electricity",
+        "rotor/diameter"
     ],
     "moreFields": [
         "{power/generator}"
     ],
     "geometry": [
-        "point",
-        "vertex",
```

Comment:
I would tend to keep `vertex` as an allowed geometry type, since a wind turbine might be mapped as being directly connected to an underground power cable for example. see #270